android/data/src/main/java/co/nimblehq/smsforwarder/data/database/FilterDto.kt

```diff
@@ -0,0 +1,21 @@
+package co.nimblehq.smsforwarder.data.database
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+internal const val SMS_FILTERING_TABLE_NAME = "Filters"
+
+@Entity(tableName = "Filters")
+data class FilterDto(
+    @PrimaryKey(autoGenerate = true)
+    val id: Int? = null,
```

Comment:
I think this should not be nullable.

Reply:
Yeah. But for now, when we are saving the Filter when don't have the id until saved successfully.